src/imperator/data/event_pictures.rs

```diff
@@ -45,3 +46,18 @@ impl DbKind for EventPicture {
         });
     }
 }
+
+pub fn verify_exists_or_empty(data: &Everything, file: &Token, max_sev: Severity) {
+    let file_str = file.as_str();
+    if file_str.is_empty() {
+        return;
+    }
+    data.fileset.mark_used(&file_str.replace("//", "/"));
+    if !data.fileset.exists(file_str) {
+        let msg = format!("file {file_str} does not exist");
+        report(ErrorKey::MissingFile, Item::File.severity().at_most(max_sev))
+            .msg(msg)
+            .loc(file)
+            .push();
```

Comment:
I'd rather not have this duplicate code. If `verify_exists_implied` changes, this would have to change too. Just calling `data.verify_exists(Item::File, file)` is better.

Reply:
The reason I added a new function for this is because using `data.verify_exists(Item::File, file)` doesn't really work here because the `icon` and `picture` fields/blocks of both event pictures and event themes allow for the field to be blank but should also still validate if the value is a valid file if it is not blank, there are several hundred instances of them being blank in Invictus so I had to roll my own slightly modified version of this to make it work. Would there be a better way to handle this? How I have it now works but I agree the duplicate code is not ideal.

Reply:
Yeah I meant that you can call `data.verify_exists(Item::File, file)` in the new function :) After checking if it's empty